.github/workflows/create-release-pr.yaml

```
@@ -55,13 +56,27 @@ jobs:
         with:
           install-nodejs: 'true'
 
+      - name: Extract the old and new versions of the package
+        id: extract-package-versions
+        run: |
+          # https://github.com/changesets/changesets/issues/1055#issuecomment-1602572294
+          npm run cs -- status --output=release.json --since=origin/main
```

```suggestion
          npm run cs -- status --output=release.json --since=origin/[current-ref]
```
should this use the current ref instead since it could be run from releases branch?